### ISSUE TITLE
Style/FormatStringToken with flags/modifiers

### ISSUE THREAD

**Issue by herwinw** [rubocop-hq/rubocop#7091]
PoC code:

```ruby
def foo
  puts format('%s %s', 'foo', 'bar')
  puts format('%-20s %s', 'foo', 'bar')
  puts format('%-20s %-30s', 'foo', 'bar')
end
```

--------

## Expected behavior

6 warnings of `Style/FormatStringToken` to prefer annotated tokens

## Actual behavior

Only the `%s` items are detected. Any flag or modifier to the format string makes them unrecognized.

## Steps to reproduce the problem

See PoC

## RuboCop version

```
0.70.0 (using Parser 2.6.3.0, running on ruby 2.3.3 i386-linux-gnu)
```

**Comment by buehmann** [comment]
I am going to take a shot at generalizing my #7268 to cover this.
